src/pip/_internal/commands/install.py

```diff
@@ -215,7 +215,8 @@ def __init__(self, *args, **kw):
             action="store_false",
             dest="warn_script_location",
             default=True,
-            help="Do not warn when installing scripts outside PATH",
+            help="Do not warn when installing scripts outside PATH. Must use"
```

Comment:
https://github.com/pypa/pip/issues/6209#issuecomment-457995970 suggested this note be added to cli help command, however it doesn't seem that necessary, as the other boolean options don't have it. thoughts?